### PR TITLE
fix broken links about declarative syntax

### PR DESCRIPTION
**Pull request by kiyonlin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 Go-app is a package for **building progressive web apps (PWA)** with the [Go programming language (Golang)](https://golang.org) and [WebAssembly (Wasm)](https://webassembly.org).
 
-Shaping a UI is done by using a **[declarative syntax](/syntax) that creates and compose HTML elements only by using the Go programing language**.
+Shaping a UI is done by using a **[declarative syntax](https://go-app.dev/#declarative-syntax) that creates and compose HTML elements only by using the Go programing language**.
```
Comment (maxence-charriere) [inline README.md:18]: The address here should point to the documentation syntax page:
```
[declarative syntax](https://go-app.dev/syntax)
```

File: README.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ go get -u github.com/maxence-charriere/go-app/v8/pkg/app
 
 ## Declarative syntax
 
-Go-app uses a [declarative syntax](/syntax) so you can **write reusable component-based UI elements** just by using the Go programming language.
+Go-app uses a [declarative syntax](https://go-app.dev/#declarative-syntax) so you can **write reusable component-based UI elements** just by using the Go programming language.
```
Comment (maxence-charriere) [inline README.md:42]: Same here.

**Review by maxence-charriere — CHANGES_REQUESTED** [review]
Hey there, thanks for this PR!
I totally forgot to update the links for the README.

The link you modified does not point to the right article though.
Let's put the right one and get this merged :).

**Comment by kiyonlin** [comment]
Done 😄

**Review by maxence-charriere — APPROVED** [review]
Thank you!

**Pull request merged**